nci/workspace_cleaner.rb

```diff
@@ -82,9 +82,22 @@ def cleanup?(workspace)
         puts '  ws-cleanup => delete'
         return true
       end
+      # delete mgmt jobs workspaces when they become old 
```

Comment:
All the new code could be moved into the `if workspace.include?('mgmt_')` below saving you the excessive checking for mgmt_.